lib/aff4.py

```diff
@@ -2393,6 +2405,10 @@ def Initialize(self):
       self.size = 0
       self.content_last = None
 
+    # This is the configurable default length for allowing Read to be called
+    # without a specific length
+    self.max_unbound_read = config_lib.CONFIG["Server.max_unbound_read_size"]
```

Comment:
What is the right way to do that?  I put it in the base class and it threw an error about config not being initialized yet.